### ISSUE TITLE
Python testcases crashes in exomerge_unit_test.py 

### ISSUE THREAD

**Issue by ac48392** [sandialabs/seacas#383]
Python testcases crashes in exomerge_unit_test.py if the deployment path of exomerge_unit_test.e is greater than seventy-nine characters.
Please find the failure log and debugging as follows,
```sh
[38] _test_export_model
Opening exodus file: temp.e
WARNING: max line length reached for one or more info records;
         info stored to exodus file is incomplete for these records
Traceback (most recent call last):
  File "/jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/tests/exomerge_unit_test.py", line 1886, in <module>
    tester. Test()
  File "/jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/tests/exomerge_unit_test.py", line 1856, in test
    if test[1]() is None:
  File "/jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/tests/exomerge_unit_test.py", line 1329, in _test_export_model
    self.model.export_model("temp.e")
  File "/jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/exomerge.py", line 7945, in export_model
    new_file.put_info_records(self.info_records)
  File "/jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/exodus.py", line 1068, in put_info_records
    return self.__ex_put_info_recs(info)
  File "/jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/exodus.py", line 4951, in __ex_put_info_recs
    str(infoRecs[i]).encode('ascii'), MAX_LINE_LENGTH + 1))
  File "/usr/lib64/python3.6/ctypes/__init__.py", line 57, in create_string_buffer
    buf.value = init
ValueError: byte string too long
<end of output>
Test time =   1.07 sec
----------------------------------------------------------
Test Failed.
"test_exomerge3.py" end time: May 13 11:07 CEST
"test_exomerge3.py" time elapsed: 00:00:01
----------------------------------------------------------

End testing: May 13 11:07 CEST
``` 

Failed analysis as follows,
```sh
(infoRecs[478]).encode(ascii): Discarded title from the following file:
and length :  40
(infoRecs[479]).encode(ascii): /jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/tests/exomerge_unit_test.e
and length :  83
(infoRecs[480]).encode(ascii): Default Sierra Title
and length :  20
```

**Comment by ac48392** [comment]
fix gives output as follows,
```sh
infoRecs[478] = Discarded title from the following file:
infoRecs[479] = /jenkins/git/workspace/DEW_components/v23.1p1/seacas/lib/tests/exomerge_unit_te\
infoRecs[480] = st.e
infoRecs[481] = Default Sierra Title
```

**Comment by gsjaardema** [comment]
Fixed by #384
